decidim-core/lib/decidim/events/simple_event.rb

```diff
@@ -34,7 +34,13 @@ def self.i18n_attributes(*attributes)
       end
 
       def email_subject
-        I18n.t("email_subject", **i18n_options).html_safe
+        I18n.t("email_subject", **updated_options).html_safe
+      end
+
+      def updated_options
```

Comment:
I think the method name is a bit unclear, so could we use instead something like `def email_subject_i18n_options`?

Reply:
Sure, I will do the change for method naming :smile: Thank you!